.github/workflows/tests.yaml

```diff
@@ -2,14 +2,14 @@ name: tests
 on: ["push", "pull_request", "workflow_dispatch"]
 
 jobs:
-  build-test:
+  run-checks:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.18', '1.17', '1.16', '1.15', '1.14', '1.13']
+        go: ['1.19']
```

Comment:
If snapd shoud consume this code we will need to include go-1.13 for now and go-1.18 later (we plan to move to that this cycle). Snapd sadly uses older go to support building on the older distro releases.

Reply:
Reverted.